### PR TITLE
fix typo in photos.json

### PR DESCRIPTION
**Pull request by ilovehans10**
#### Summary
None

#### Purpose of change
This fixes a typo added in the new photos.json.

#### Describe the solution
Replace formerly with formally, so the woman is no longer formerly dressed.

#### Additional context
This is my first PR on github, so if I did something wrong please let me know. Also I know this is really small for a pull request, but I just saw this and figured it would be a quick fix and a good way to finally rip the bandaid off as far as contributing goes.

### CODE REVIEW

**Review by Maleclypse — APPROVED** [review]
Looks good! Thanks for contributing!

**Comment by ilovehans10** [comment]
I have updated dictionary.txt to include kerbside as it is a word, but this is apparently the spelling used by our friends across the pond. I am unsure if American English is the preferred spelling for this project, or if just any generic English works. If we should try to only use American English, I can quick make that fix.

Other than that, I believe that this is ready to be pulled if it is alright with maintainers. Is there a specific way that I should request review in the future.

**Comment by ilovehans10** [comment]
Ok, in the future I will probably not do separate commits for each little change, but it's been a while since I last used git. I think I am ready for good this time, unless spellcheck finds some more things for me to fix.

File: data/json/snippets/photos.json [hunk 1]
```diff
@@ -134,11 +134,11 @@
...
...e was a living room: the window blinds thickly drawn and boards nailing down any cracks through which external observation could be possible.  The entire space looks alive, with some manner of spongy vegetation coating its surfaces and thick tendrils and quivering mounds of mould subsuming most of the furniture.  Reclined within the misshapen gray lump that might have been a sofa, a pale-skinned woman lounges, her spine arched, head thrown back, and a look of rapturous ecstasy visible on the parts of her face not caked by the fungus that’s extending in wet stalks from her open mouth.  Smiling contentedly, with dainty tendrils spreading across her face, entwining in her hair, and burrowing into the ruptured sockets where eyes used to reside, the woman is embracing a bulging cocoon of gray resting in her lap, small bits of fur peeking through cracks in the fungal shell.  In bold and crude calligraphy, text has been scrawled on the back of the image.\n\n\"Do not be afraid, for our beauty is all-encompassing and rests beyond your sight.\"\n\n\"Do not be afraid, for we speak the word of blissful hegemony.\"\n\n\"Do not be afraid, for this is our mission, and you will be a part of it.\""
```
Comment (I-am-Erk) [inline data/json/snippets/photos.json:141]:
```suggestion
        "text": "Ragged, unclear, and altogether hurriedly developed, this photograph appears to depict the gloomy interior of what once was a living room, with window blinds thickly drawn and boards nailing down any cracks through which external observation could be possible.  The entire space looks alive: some manner of spongy vegetation coats its surfaces and thick tendrils and quivering mounds of mold subsume most of the furniture.  Reclined within the misshapen gray lump that might have been a sofa, a pale-skinned woman lounges, her spine arched, head thrown back, and a look of rapturous ecstasy visible on the parts of her face not caked by the fungus that’s extending in wet stalks from her open mouth.  Smiling contentedly, with dainty tendrils spreading across her face, entwining in her hair, and burrowing into the ruptured sockets where eyes used to reside, the woman is embracing a bulging cocoon of gray resting in her lap, small bits of fur peeking through cracks in the fungal shell.  In bold and crude calligraphy, text has been scrawled on the back of the image.\n\n\"Do not be afraid, for our beauty is all-encompassing and rests beyond your sight.\"\n\n\"Do not be afraid, for we speak the word of blissful hegemony.\"\n\n\"Do not be afraid, for this is our mission, and you will be a part of it.\""
```

**Comment by I-am-Erk** [comment]
A few more little ones ("mould" is also a correct spelling eschewed by americans) and cleared up a somewhat awkward wording.

It's probably good the woman is dressed again.

I'll merge as soon as the tests allow me

**Pull request merged**